v2/api-validator/tests/server-tests/collateral/deposit.test.ts

```diff
@@ -15,31 +19,63 @@ import config from '../../../src/config';
 import Client from '../../../src/client';
 
 const noCollateralCapability = !hasCapability('collateral');
+const noTransferCapability = !hasCapability('transfers');
 
-describe.skipIf(noCollateralCapability)('Collateral Deposit', () => {
+describe.skipIf(noCollateralCapability || noTransferCapability)('Collateral Deposit', () => {
   const client: Client = new Client();
   const collateralId: string = config.get('collateral.collateralAccount.accountId');
   const fireblocksIntentId = uuid();
-  let assetId: string;
   let accountId: string;
 
+  function isNativeCryptocurrency(
+    asset: NativeCryptocurrency | OtherAssetReference | NationalCurrency
+  ): asset is NativeCryptocurrency {
+    return typeof asset === 'object' && 'cryptocurrencySymbol' in asset && 'blockchain' in asset;
+  }
+
+  function isOtherAssetReference(
+    asset: NativeCryptocurrency | OtherAssetReference | NationalCurrency
+  ): asset is OtherAssetReference {
+    return typeof asset === 'object' && 'assetId' in asset;
+  }
+
   beforeAll(async () => {
     accountId = getCapableAccountId('collateral');
-    const assetsResult = await client.capabilities.getAdditionalAssets({});
-    assetId = assetsResult.assets[0]?.id;
   });
 
   describe('Register collateral deposit transaction (add collateral) & fetch by collateralTxId', () => {
+    const deposits: PublicBlockchainAddress[] = JSON.parse(config.get('collateral.deposit.assets'));
     const collateralTxId = `2.${uuid()}.${collateralId}`;
     const intentApprovalRequest: IntentApprovalRequest = { fireblocksIntentId: fireblocksIntentId };
     let partnerIntentId: string;
-    it('Initiate should return valid response', async () => {
-      const asset: CryptocurrencyReference = { assetId: assetId };
+    it.each(deposits)('Initiate should return valid response', async (testParams) => {
```

Comment:
Looks like something like `depositDestination` would be more descriptive than `testParams`.